src/simplay/src/simplay/components.py

```diff
@@ -1,18 +1,15 @@
 from typing import Any, Callable, Union
-from simpy import (
-    Resource,
-    PreemptiveResource,
-    PriorityResource,
-    Container,
-    Store,
-    FilterStore,
-)
-
-from simpy.resources.resource import Request, Release, PriorityRequest
-from simpy.resources.container import ContainerAmount, ContainerPut, ContainerGet
-from simpy.resources.store import StoreGet, StorePut, FilterStoreGet
-from simplay.core import VisualEnvironment, VisualComponent
-from simplay.visualutil import ResourceVisualUtil, ContainerVisualUtil, StoreVisualUtil
+
+from simpy import (Container, FilterStore, PreemptiveResource,
+                   PriorityResource, Resource, Store)
+from simpy.resources.container import (ContainerAmount, ContainerGet,
+                                       ContainerPut)
+from simpy.resources.resource import PriorityRequest, Release, Request
+from simpy.resources.store import FilterStoreGet, StoreGet, StorePut
+
+from .core import VisualComponent, VisualEnvironment
+from .visualutil import (ContainerVisualUtil, ResourceVisualUtil,
+                         StoreVisualUtil)
 
 
 class VisualProcess(VisualComponent):
```

Comment:
I agree with you, and where we come from within software engineering that is usually the case. Unfortunately, it is not the python way of life. I've already scrambeled to separate the code as much as possible. Since these classes will mostly be used alongside each other, separating them would lead to a separate import statement for each class, thus adding indirection where not is needed IMO